app/TipsDetails.tsx

```diff
@@ -0,0 +1,115 @@
+import { Dimensions, ScrollView, StyleSheet, Text, View } from 'react-native';
+
+import { TIPS_INFO } from './data/TipsInfo';
+import LightbulbIcon from './icons/LightbulbIcon';
+
+const screenHeight = Dimensions.get('window').height;
```

Comment:
you could do const { width: screenWidth, height: screenHeight } = Dimensions.get('window');